fe/fe-core/src/main/java/com/starrocks/leader/ReportHandler.java

```diff
@@ -456,6 +479,24 @@ private static void workgroupReport(long backendId, List<TWorkGroup> workGroups)
                 backendId, System.currentTimeMillis() - start, workGroups.size());
     }
 
+    // For test.
+    public static void testHandleResourceUsageReport(long backendId, TResourceUsage usage) {
+        resourceUsageReport(backendId, usage);
+    }
+
+    private static void resourceUsageReport(long backendId, TResourceUsage usage) {
+        LOG.debug("begin to handle resource usage report from backend {}", backendId);
+        long start = System.currentTimeMillis();
+        boolean isChanged = QueryQueueManager.getInstance().updateResourceUsage(
+                backendId, usage.getNum_running_queries(), usage.getMem_limit_bytes(), usage.getMem_used_bytes(),
+                usage.getCpu_used_permille());
+        if (isChanged) {
```

Comment:
If the times of updateResourceUsage's invocations here want to be suppressed, It would be better to use stddev/variance to express a negligible extent in which the resource usage need not to be updated.

Reply:
Maybe it is OK, since the FE leader only send one RPC to each FE follower every second.
And FE follower needs know the instant resource usage fro every second.